queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT NAME from animals WHERE name LIKE '%mon%';
```

Comment:
- [ ] Consider listing all animals whose name ends in "mon" and not contain mon, so it will be like this `SELECT * FROM animals WHERE name LIKE '%mon';`

Reply:
@omarsalem7 Thank you very much for pointing this out and guiding me.
I have corrected myself.

here is the screenshot.
![image](https://user-images.githubusercontent.com/40719899/196225935-ede4bbc5-1bb0-400a-b63a-e8619b1d1ae6.png)